content/en/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function.md

```diff
@@ -145,9 +145,20 @@ If you are collecting logs from a CloudWatch log group, configure the trigger to
 For Terraform users, you can provision and manage your triggers using the [aws_cloudwatch_log_subscription_filter][1] resource. See sample code below.
 
 ```conf
+data "aws_cloudwatch_log_group" "some_log_group" {
+  name = "/some/log/group"
+}
+
+resource "aws_lambda_permission" "lambda_permission" {
+  action        = "lambda:InvokeFunction"
+  function_name = "datadog-forwarder" # this is the default but may be different in your case
+  principal     = "logs.${data.aws_region.current.name}.amazonaws.com"
```

Comment:
should match the principal set in https://github.com/DataDog/datadog-serverless-functions/blob/master/aws/logs_monitoring/template.yaml#L682 

it's either `logs.amazonaws.com` or `logs.amazonaws.com.cn` for china